templates/config.tfvars.openstack-template

```diff
@@ -19,10 +19,14 @@ master_as_edge = "true"
 # Node configuration
 node_count = "3"
 node_flavor = "your-node-flavor"
+# node_assign_floating_ip = "true"
+# node_extra_disk_size = "10"
 
 # Edge configuration
 # edge_count = "2"
 # edge_flavor = "your-edge-flavor"
+# To disable traefik in favor for another ingress controller comment out the line below.
+ingress = "traefik"
```

Comment:
Isn't it enough to have edge_count to 0, to skip Traefik deployment? I think that we use node labels, so if you don't have edges and set master_as_edge to false it should not be deployed.

Reply:
Ok I see! What do you use instead?

Reply:
nginx